packages/core/src/integrations/supabase.ts

```diff
@@ -236,6 +236,7 @@ function instrumentAuthOperation(operation: AuthOperationFn, isAdmin = false): A
                 captureException(res.error, {
                   mechanism: {
                     handled: false,
+                    type: 'auto.db.supabase.auth',
```

Comment:
these types are one level more specific than the trace origin on the spans (`auto.db.supabase`). I think this is fine.